x-pack/plugins/apm/server/routes/services/get_service_instances/get_service_instances_transaction_statistics.ts

```diff
@@ -48,6 +49,20 @@ type ServiceInstanceTransactionStatistics<T> = T extends true
   ? ServiceInstanceTransactionComparisonStatistics
   : ServiceInstanceTransactionPrimaryStatistics;
 
+export function getOrderInstructions(
+  sortField: Exclude<InstancesSortField, 'errorRate'>,
+  sortDirection: 'asc' | 'desc'
+): Record<string, 'asc' | 'desc'> {
+  switch (sortField) {
+    case 'latency':
```

Comment:
I might be missing something here, but Latency sub agg runs `avg` or `percentile` aggregations. According to the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#_ordering_by_a_sub_aggregation), sorting the terms aggs by it might produce wrong results.

Reply:
I don't think there's much we can do in this case.